install.sh

```diff
@@ -7,7 +7,13 @@ conda install -c anaconda pip
 # Install requirements
 pip install -r requirements.txt
 
+# Install pytorch to support GPU on mac m1
+# Expect the M1-GPU support to be included in the next stable release. For the time being, it only is found in the Nightly release.
+conda install pytorch -c pytorch-nightly
+
 # Install ray
+# Firts make sure grpcio is only installed with conda
+pip uninstall grpcio; conda install grpcio=1.43.0
```

Comment:
what is it needed for?

Reply:
why does it need to be installed through conda and can't be specified in the requirements as grpcio==1.43.0?